### PR TITLE
Hide image on mobile landscape

### PR DESCRIPTION
**Pull request by fenglish**
local ft com_8080_vanity-test-article-22cda1acxd2a0x42ecx928axf26f6b5c34b6(iPhone X)

### CODE REVIEW

File: components/x-audio/src/components/styles.scss [hunk 1]
```diff
@@ -359,3 +358,36 @@ $primary-scrubber-color: oColorsGetPaletteColor('teal');
 // 	width: var(--buffered-width);
 // 	background-color: green;
 // }
```
Comment (benbarnett) [inline components/x-audio/src/components/styles.scss]: Oh I know this change is nothing to do with this PR but I think this code was accidentally committed. That's right isn't it @robsquires? I think this was an experiment for buffering stuff that we're not doing anymore.

File: components/x-audio/src/components/styles.scss [hunk 2]
```diff
@@ -359,3 +358,36 @@ $primary-scrubber-color: oColorsGetPaletteColor('teal');
...
+
+@media (orientation: landscape) and (max-height: 620px - 1) {
```
Comment (benbarnett) [inline components/x-audio/src/components/styles.scss]: Just looking at this, I suppose the actual issue here is just the `max-height`, not necessarily the orientation. Is that right? Like if you shrank your browser down to a small height, you'd probably also want these rules to apply. What do you think?

Comment (fenglish) [reply components/x-audio/src/components/styles.scss]: @benbarnett That's true. I'm going to remove the landscape query. Thank you!

File: components/x-audio/src/components/styles.scss (hunk 1 above)
Comment (robsquires) [reply components/x-audio/src/components/styles.scss]: yup totally, good spot!

**Pull request merged**